examples/basic-infection/README.md

```diff
@@ -296,20 +297,20 @@ When all plans have executed for a given time t, `ixa` calls the on_period_end
 callback to write the report row and schedule the next periodic report:
 
 ```rust
-fn report_periodic_item(day, context) {
+fn report_periodic_item(t, context) {
     // Add a row for each status
     for infection_status in counter::iter() {
         context.add_report(Period(
-            day,
+            t,
             infection_status,
             count: counter::get_count(infection_status)
         ));
     }
 
-    // Schedule next report day
+    // Schedule next report
     next_report_time = context::get_time() + parameters::get_parameter(reporting_period);
-    if next_report_time < parameters::get_parameter(max_days) {
-        context::plans_did_execute(report_periodic_item, report_periodic_item);
+    if next_report_time < parameters::get_parameter(max_periods) {
```

Comment:
I don't think `max_periods` makes sense here.  My initial thought here was that if there is a maximum time specified by the user (e.g., 100 days), then reports should stop scheduling a next report time after 100 days. But that would mean that every time a periodic report is included, a `max_time` should also be specified.   

My understanding is that the context will execute until there are no more plans scheduled, but if `max_time` isn't specified, reports continue to be scheduled indefinitely. I didn't address this in my example, but I think it's important to determine **when should reports stop?**